tests/contracts/root_chain/test_start_standard_exit.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+from ethereum.tools.tester import TransactionFailed
+
+from plasma_core.constants import NULL_ADDRESS, NULL_ADDRESS_HEX, WEEK
+
+
+def test_start_standard_exit_should_succeed(testlang, utxo):
+    testlang.start_standard_exit(utxo.owner, utxo.spend_id)
+    assert testlang.root_chain.exits(utxo.spend_id) == [utxo.owner.address, NULL_ADDRESS_HEX, utxo.amount]
+
+
+def test_start_standard_exit_twice_should_fail(testlang, utxo):
+    testlang.start_standard_exit(utxo.owner, utxo.spend_id)
+    with pytest.raises(TransactionFailed):
+        testlang.start_standard_exit(utxo.owner, utxo.spend_id)
+
+
+def test_start_standard_exit_invalid_proof_should_fail(testlang, utxo):
+    spend_tx = testlang.child_chain.get_transaction(utxo.spend_id)
+    with pytest.raises(TransactionFailed):
+        testlang.root_chain.startExit(utxo.spend_id, spend_tx.encoded, b'')
```

Comment:
RootChain.startExit takes 4 arguments. Here we provide just 3. How does that work?